smdebug/tensorflow/base_hook.py

```diff
@@ -4,6 +4,7 @@
 from typing import List, Set
 
 # Third Party
+import tensorflow as tf
```

Comment:
reason for not import tensorflow.compat.v1 as tf 
All places above in PR you have replaced import tensorflow as tf with import tensorflow.compat.v1 as tf

Reply:
updated in latest commit